src/debugAdapter/goDebug.ts

```diff
@@ -280,7 +312,10 @@ class GoDebugSession extends DebugSession {
 
 	protected launchRequest(response: DebugProtocol.LaunchResponse, args: LaunchRequestArguments): void {
 		// Launch the Delve debugger on the program
-		this.delve = new Delve(args.mode, args.program, args.args, args.cwd, args.env, args.buildFlags, args.init);
+		let remotePath = typeof args.remotePath === 'undefined' ? "":args.remotePath;
```

Comment:
Just for consistency, the codebase typically just uses `args.remotePath || ""` style.